README.rst

```diff
@@ -445,6 +443,51 @@ A few builtin functions are listed in ``simpleeval.DISALLOW_FUNCTIONS``.  ``type
 If you need to give access to this kind of functionality to your expressions, then be very
 careful.  You'd be better wrapping the functions in your own safe wrappers.
 
+There is an additional layer of protection you can add in by passing in ``allowed_attrs``, which
+makes all attribute access based opt-in rather than opt-out - which is a lot safer design:
+
+.. code-block:: pycon
+
+    >>> simpleeval("' hello   '.strip()", allowed_attrs={})
```

Comment:
The `{}` syntax here breaks with the type that `allowed_attrs` would normally be -- a set.

```suggestion
    >>> simpleeval("' hello   '.strip()", allowed_attrs=set())
```

Reply:
`allowed_attrs` is a dict with type keys & set values, isn't it? `Dict[Type, Set]` so an empty list should be the correct type at this point?

Reply:
Ah, yeah, I think you're right.